### PR TITLE
API documentation for setDisplayFrequency() and getDisplayFrequencies().

### PR DESCRIPTION
**Pull request by sgosselin**

### CODE REVIEW

File: api/init.lua [hunk 1]
```diff
@@ -15979,6 +15999,33 @@ return {
...
+              returns = {
+                {
+                  name = "success",
+                  type = "boolean",
+                  description = "Whether the display refresh rate was successfully set."
+                }
+              }
+            }
+          },
+          notes = "Changing the display refresh-rate also changes the frequency of lovr.update() and lovr.draw() as they depend on the display frequency."
```
Comment (sgosselin) [inline api/init.lua:16027]: I'm not sure whether it's worth adding; let me know if you think we should just remove this.

**Comment by bjornbytes** [comment]
Thanks for updating the docs!  However, this change was made in the autogenerated file with everything merged together.  The changes won't persist there unless there is a corresponding source file in `api/lovr/headset/getDisplayFrequencies.lua` etc.

**Comment by sgosselin** [comment]
@bjornbytes ah, gotcha! I'll move the bits into their own file. How do I re-generate api/init.lua for testing?

**Comment by bjornbytes** [comment]
You can do `lovr api` to regenerate it

**Comment by sgosselin** [comment]
My bad, I just noticed it was written in the README. I'm not sure, should I regenerate the api/init.lua in this PR?

**Comment by bjornbytes** [comment]
Either way is okay.  It gets regenerated quite often.

**Pull request merged**